quicksilver/versions.json

```diff
@@ -434,7 +434,7 @@
         "v1.6.1"
       ],
       "binaries": {
-        "linux/amd64": "https://github.com/ingenuity-build/quicksilver/releases/download/v1.6.1/quicksilverd-v1.6.1-amd64"
+        "linux/amd64": "https://github.com/quicksilver-zone/quicksilver/releases/download/v1.6.1/quicksilverd-v1.6.1-amd64"
```

Comment:
Note that it is recommended (and may soon be updated automatically) to combine all patch and minor versions into a single object representing the major version (in this case, v1 would include v1.6.1, v1.6.3, v1.7.5, etc. as compatible versions with each other).

Reply:
Quicksilver hasn't been applying Semver correctly (fixing this soon!), but these versions are not compatible.

I appreciate this is on us to resolve, but won't this screw up all historic versions (and for other chains not using semver accurately?)

Reply:
Thanks for explaining. Yes, blindly assuming semver has been followed correctly for all chains, and then combining versions as if it were, could mess up version histories, and likely cannot be done safely. However, I'd at least opportunistically ask about cases where I suspect it may possible to clean up a version history. Since these patch versions are not compatible, we can keep these separated as is.